docusaurus/docs/Android/04-compose/05-message-components/08-reactions-picker.mdx

```diff
@@ -87,7 +84,7 @@ Next, let's see how to customize `ReactionsPicker`.
 @Composable
 public fun ReactionsPicker(
     ..., // State,
-    reactionTypes: Map<String, Int> = ChatTheme.reactionTypes,
+    reactionTypes: Map<String, ReactionIcon> = ChatTheme.reactionIconFactory.createReactionIcons(),
     ... // Actions and content
 )
 ```
```

Comment:
`By default it uses reactionIconFactory inside of...`

`You can customize these either by overriding ChatTheme.reactionIconFactory`